includes/class-donations.php

```diff
@@ -1094,5 +1086,22 @@ public static function disable_coupons( $enabled ) {
 		}
 		return false;
 	}
+
+	/**
+	 * Prevent rendering of Donate block if Reader Revenue platform is set to 'other.
+	 *
+	 * @param string $block_content The block content about to be rendered.
+	 * @param array  $block The data of the block about to be rendered.
+	 */
+	public static function prevent_rendering_donate_block( $block_content, $block ) {
+		if (
+			isset( $block['blockName'] )
+			&& 'newspack-blocks/donate' === $block['blockName']
+			&& self::is_platform_other()
+		) {
+			return '';
+		}
+		return $block_content;
+	}
```

Comment:
I moved this from the wizard class because it's not relevant for the wizard, it makes more sense to live in the more general Donations class.